README.md

```diff
@@ -146,6 +146,14 @@ Autoload can be customised using the following options:
     forceESM: true
   })
   ```
+- `shareContext` (optional) - Defaults to 'false', if set to 'true' each plugin loaded is wrapped with [fastify-plugin](https://github.com/fastify/fastify-plugin). This allows you to share contexts between plugins and the parent context if needed. For example, if you need to share decorators. Read [this](https://github.com/fastify/fastify/blob/main/docs/Reference/Encapsulation.md#sharing-between-contexts) for more details.
```

Comment:
I prefer an `encapsulate` flag that defaults to `true`.

Reply:
that's funny, I did exactly that and called this `encapsulated` at first then changed it since I thought `shareContext` would be more explicit user friendly. If you feel strongly about it, happy to change.

Reply:
In the fastify doc we speak about `context` in the encapsulation part with the doc you linked: https://github.com/fastify/fastify/blob/main/docs/Reference/Encapsulation.md#sharing-between-contexts
Mainly the feature is called encapsulation. I join @mcollina on this point, we should uniformize the wording.

Reply:
Fair, I get that. Will do

Reply:
@mcollina @zekth done!